packages/error-reporting-service/src/error-reporting-service.ts

```diff
@@ -99,23 +95,43 @@ type ErrorReportingServiceOptions = {
  * import { ErrorReportingService } from '@metamask/error-reporting-service';
  * import { ExampleController } from './example-controller';
  *
+ * type AllActions = MessengerActions<ErrorReportingServiceMessenger>;
+ *
+ * type AllEvents = MessengerEvents<ErrorReportingServiceMessenger>;
+ *
+ * type RootMessenger = Messenger<'Root', AllActions, AllEvents>;
+ *
  * // Create a global messenger.
  * const globalMessenger = new Messenger();
```

Comment:
Maybe we can simplify this a bit? Also in this example should we go with global messenger instead of root messenger?

``` suggestion
 * type GlobalMessenger = Messenger<
 *   'Global',
 *   MessengerActions<ErrorReportingServiceMessenger>,
 *   MessengerEvents<ErrorReportingServiceMessenger>
 * >;
 *
 * // Create a global messenger.
 * const globalMessenger: GlobalMessenger = new Messenger();
```